src/node.cc

```diff
@@ -3334,6 +3338,13 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "_debugWaitConnect", True(env->isolate()));
   }
 
+  // --expose_internals, --expose-internals
+  // Note that this is not exposed as a process property, it is deleted when
```

Comment:
I didn't think it was necessary either, but it confused @jasnell - James, what do you think?